messages.xml

```diff
@@ -303,20 +303,22 @@
     <slot weight="1">
       <contact>atong@cloudbees.com</contact>
       <message>
-        <caption>Templating Jenkins Environments with Docker</caption>
-        <link>https://www.cloudbees.com/blog/templating-jenkins-build-environments-docker-containers</link>
-        <blurb>Read this Blog to Learn How</blurb>
-        <logo>https://jenkins.io/images/patrons/cloudbees.png</logo>
+        <caption>Become Jenkins Certified</caption>
+        <link>https://www.cloudbees.com/jenkins-certification</link>
+        <blurb>The first certification program for Jenkins Engineers.</blurb>
+        <logo>https://pages.cloudbees.com/rs/083-PKZ-512/images/ci_cert1.png</logo>
+        <!-- TODO to be copied over to jenkins.io for the duration of this message before publishing -->
```

Comment:
Right, that I'm definitely in favor of doing. We should update them all at once IMO